### ISSUE TITLE
single sample burst not handled in readECOraw.m and readBB9raw.m

### ISSUE THREAD

**Issue by sspagnol** [aodn/imos-toolbox#503]
Similar to fix in readWQMdat.m, calculation of durationBurst etc require

```matlab
    for i=1:nBurst
        timeBurst = time(iBurst(i):iBurst(i+1)-1);
        sampleIntervalInBurst(i) = median(diff(timeBurst*24*3600));
        firstTimeBurst(i) = timeBurst(1);
        durationBurst(i) = (timeBurst(end) - timeBurst(1))*24*3600 + sampleIntervalInBurst(i);
    end
```
changed to
```matlab
  for i=1:nBurst
      timeBurst = time(iBurst(i):iBurst(i+1)-1);
      if numel(timeBurst)>1 % deals with the case of a file with a single sample in a single burst
          sampleIntervalInBurst(i) = median(diff(timeBurst*24*3600));
          firstTimeBurst(i) = timeBurst(1);
          durationBurst(i) = (timeBurst(end) - timeBurst(1))*24*3600 + sampleIntervalInBurst(i);
      end
  end
```

**Comment by ggalibert** [comment]
Sounds good, will include in next version.

**Comment by ggalibert** [comment]
Included in v2.5.39 via https://github.com/aodn/imos-toolbox/commit/fe0d4461caed55e4c736ac0c954e57d59d3f4096
